usecases/auth/authorization/rbac/types.go

```diff
@@ -77,7 +77,7 @@ func permission(policy []string) *models.Permission {
 	action := fmt.Sprintf("%s_%s", authorization.Actions[policy[2]], domain)
 	if domain == "objects" {
 		action += "_collection"
-	} else if domain == "tenants" {
+	} else if domain == "objects_tenant" {
```

Comment:
**issue:**  that domain doesn't exists 🤔

Reply:
Hmm, actually you're correct. That will just return both as `objects` 🤔 There needs to be a distinction between `objects_collection` and `object_tenants` at this level though

Reply:
fixed!